tests/queries/0_stateless/00652_mutations_alter_update.sh

```diff
@@ -123,5 +123,46 @@ ${CLICKHOUSE_CLIENT} --query="SELECT value2, materialized_value FROM alter_updat
 
 ${CLICKHOUSE_CLIENT} --query="ALTER TABLE alter_update DROP PARTITION 200001"
 
+${CLICKHOUSE_CLIENT} --query="SELECT '*** Test materialized column can be updated with SETTINGS update_allow_materialized_columns=1 ***'"
```

Comment:
A test with ReplicatedMergeTree is required as well

Reply:
Fixed in the latest commit.